doc/rst/language/evolution.rst

```diff
@@ -8,6 +8,231 @@ capture significant language changes that have the possibility of breaking
 existing user codes or code samples from old presentations or papers that
 predated the changes.
 
+Note that the compiler flag ``--warn-unstable`` is available and can be
+useful when migrating programs to the current version of the language.
+The purpose of this flag is to identify  portions of a program that use a
+language feature that has changed meaning.
+
+version 1.18, September 2018
+----------------------------
+
+Version 1.18 includes many language changes that address problems with
+classes.
+
+In summary:
+
+ * constructors are deprecated and replaced with initializers
+   See :ref:`readme-evolution.initializers-replace-constructors`
+ * memory management for class types has changed
+   See :ref:`readme-evolution.class-memory-management`
+ * `override` is now required on overriding methods
+   See :ref:`readme-evolution.mark-overriding`
+
+.. _readme-evolution.initializers-replace-constructors:
+
+initializers replace constructors
+*********************************
+
+Code that was contained user-defined constructors will need to be updated
+to use an initializer. For example:
+
+.. code-block:: chapel
+
+  record Point {
+    var x, y: real;
+    proc Point() {
+      x = 0;
+      y = 0;
+      writeln("In Point()");
+    }
+    proc Point(x: real, y: real) {
+      this.x = x;
+      this.y = y;
+      writeln("In Point(x,y)");
+    }
+  }
+  var a:Point;
+  var b = new Point(1.0, 2.0);
+
+will now compile with deprecation warnings. Here is the same program
+updated to use initializers:
+
+.. code-block:: chapel
+
+  record Point {
+    var x, y: real;
+    proc init() {
+      x = 0;
+      y = 0;
+      writeln("In Point.init()");
+    }
+    proc init(x: real, y: real) {
+      this.x = x;
+      this.y = y;
+      writeln("In Point.init(x,y)");
+    }
+  }
+  var a:Point;
+  var b = new Point(1.0, 2.0);
+
+
```

Comment:
At a glance, this example just looks like changing the name of the method. I don't think users will assume that's the extent of the change, but they might be left wanting more.

Should we say something about how there are a number of different semantics for initializers, and that the user should read the language spec for more info?